tensorboard/uploader/proto/write_service.proto

```diff
@@ -128,3 +138,37 @@ message DeleteOwnUserRequest {
 message DeleteOwnUserResponse {
   // This is empty on purpose.
 }
+
+
+// Request to change the name of one experiment.
+message UpdateExperimentMetadataRequest {
+  Experiment updates = 1;
+  ExperimentMetadataMask update_mask = 2;
+}
+
+message ExperimentMetadataMask {
```

Comment:
Naming nit: ExperimentMetadataMask does not have an obviously different meaning from ExperimentMask.  How about ExperimentUpdateMask?

Reply:
Or UpdateExperimentMask, etc.

Reply:
Yes, thank you—it should be `ExperimentMask`. (Vestige of earlier draft
of comment.)